the-telephone-app-fe/src/components/ShowResult/ShowResult.js

```diff
@@ -16,14 +16,19 @@ const ShowResult = () => {
     navigate("/");
   };
 
-  const [currentTurn, setCurrentTurn] = useState(0);
+  const location = useLocation();
+  const [users, setUsers] = useState(location.state?.data);
+  const role = location.state?.role;
+  const resultSet = location.state?.dataReceive;
+  const [playerName, setPlayerName] = useState();
+  let [player, setPlayer] = useState(0);
+  const [results, setResult] = useState([]);
+  const [timer, setTimer] = useState(0);
+  const scrollPoint = useRef(null);
+  const scrollPoint2 = useRef(null);
 
   const nextPlayer = () => {
     setPlayer((player += 1));
```

Comment:
the player is supposed to update only through setPlayer, so is this should just be player+1 instead of assigning with += ?